content/sips/sip-229.md

```diff
@@ -51,18 +53,17 @@ The `optimism.github.io` repository will need to be updated to include the `sUSD
 
 #### Chainlink
 
-Chainlink will need to track a new source for the issued synths and debt ratio oracle, the synths in transit.
-
-In an atomic call, the oracle should retrieve the values for `currentDebt` and the last `n` events for `SynthSent`, and the last `n` events for `SynthReceived`, where `n` is the largest expected number of synth transfers in flight.
+Chainlink will need to track 2 new sources per chain for the issued synths and debt ratio oracle, the synths in transit.
 
-After retrieving data from all networks, the synths in transit can be calculated:
+In pseudocode, the new equation for the `issued synths` value within the debt oracles is:
 
 ```
-let synthsNotReceivedEvents = synthSentEvents.filter(e => synthReceivedEvents.find(e.id))
-let synthsInTransit = sum(synthsNotReceivedEvents, 'amount');
+sum(DebtCache.currentDebt() + SynthetixBridge.synthTransferReceived() - SynthetixBridge.synthTransferSent()) for all chains
```

Comment:
Maybe better to provide a single view for them to make possible future changes easier to integrate. Something like `sumChainDebts` or semething.